roles/kubernetes-apps/ansible/defaults/main.yml

```diff
@@ -22,7 +22,7 @@ exechealthz_image_tag: "{{ exechealthz_version }}"
 calico_policy_controller_cpu_limit: 100m
 calico_policy_controller_memory_limit: 256M
 calico_policy_controller_cpu_requests: 30m
-calico_policy_controller_memory_requests: 128M
+calico_policy_controller_memory_requests: 64M
```

Comment:
It looks good to me. However, I think it's a time to start thinking in terms of % (calculated numbers). In this case we can utilise resources optimally.